### DISCUSSION TITLE
Post based on developer's skill level or seniority

### DISCUSSION THREAD

**Discussion by Dubask** [dailydotdev/daily#966] [Feature Requests]
Hi,
LOVE your product.
But I keep stumbling upon posts with great headlines but that is really for beginners (not fundamentals, just beginners).
I will be of great value if you can develop some labeling system that can detect if a post is for juniors, seniors, etc...
And then to be able to choose in the preferences post that are directed for a certain skill level or seniority.

Thanks!

**Comment by idoshamun** [comment]
Hi @Dubask,

This is a serious issue we're trying to tackle.
It has some non-trivial challenges, but it's our focus to improve our recommendations and provide better control for our users
